doc/source/contribution/documentation/i18n.rst

```diff
@@ -26,12 +26,13 @@ Here is a translation flow:
 
 1. Install Sphinx, if it is not installed.
 2. Clone Groonga repository.
-3. Update .po files.
-4. Edit .po files.
-5. Generate HTML files.
-6. Confirm HTML output.
-7. Repeat 2.-4. until you get good result.
-8. Send your works to us!
+3. Update .edit files
+4. Edit .edit files
+5. Update .po files.
+6. Generate HTML files.
+7. Confirm HTML output.
+8. Repeat 3.-7. until you get good result.
```

Comment:
```suggestion
8. Repeat 3.-7. until you finish to translate.
```

to align with the previous wording.

Reply:
Ah, maybe you see the old diff. It had been fixed in https://github.com/groonga/groonga/pull/1407/commits/5bd6dc43776288a58f362287a91a2432bfd6c3cf (the latest commit).